scripts/nettraffic

```diff
@@ -1,27 +1,43 @@
 #!/bin/sh
 
+# span for font
+fspan(){
+    echo "<span font_desc=\"${VALUE_FONT}\" color=\"${VALUE_COLOR}\"> ${1}</span>"
+}
+
+# span for label/icon
+lspan(){
+    echo "<span color=\"${LABEL_COLOR}\"> ${1}</span>"
```

Comment:
The spans for "icon fonts" should not use the space character as that maps to something other than space in some font character sets.  Here is what this version of nettraffic looks like with materialdesign instead of fontawesome:
![image](https://user-images.githubusercontent.com/49683/69554848-3db40200-0f57-11ea-8c04-c78a377fae5b.png)

but I change the expression to `echo "<span color=\"${LABEL_COLOR}\">${1}</span>"` there are no garbage characters.

Reply:
Oh, I see. I was wondering about that. Great!